pylua/network.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+
+import asyncio
+import websockets
+from websockets import connect
+
+async def server_task(websocket, path): #pylint: disable=unused-argument
```

Comment:
@OHerasym, avoid suppression, remove path, or if it's required because this is callback, then find the way to use some better suppression: there is a way to rename unused param to "dummy_path" or something like this.